demos/polls/aiohttpdemo_polls/db.py

```diff
@@ -1,6 +1,7 @@
-import aiopg.sa
 import sqlalchemy as sa
 
+import aiopg.sa
```

Comment:
when I run the tests locally flake said to run isort against this file, still want me to revert?